i18n/locales/portuguese/translations.json

```
@@ -11,7 +11,11 @@
     "see-results": "Ver todos os resultados para {{ searchQuery }}",
     "no-tutorials": "Não foram encontrados tutoriais"
   },
-  "banner": "<span>Converse com outros desenvolvedores em português</span>",
+  "banner": {
+    "default": "<span>Converse com outros desenvolvedores em português</span>",
```

Wow, didn't catch this existing issue. If you don't mind, we can fix it in this PR:

```suggestion
    "default": "{{ <0> }}Converse com outros desenvolvedores em português{{ </0> }}",
```